Makefile

```diff
@@ -118,7 +118,6 @@ test:
 	hack/test-cmd.sh
 	KUBE_RACE=" " hack/test-integration-docker.sh
 	hack/test-end-to-end-docker.sh
-	hack/test-end-to-end.sh
```

Comment:
nm, I see... I feel like test-end-to-end will atrophy... not sure what to do about that